### ISSUE TITLE
No Dates in Folders

### ISSUE THREAD

**Issue by ZL1LAC** [JVital2013/goestools-win#6]
I noticed the folders are not dated, like in goesproc for Linux, is this intentional?

**Comment by JVital2013** [comment]
Goestools for Windows does support dated subfolders just like Goestools for Linux - it's just that the default config on the Windows side doesn't come configured for it. The `directory` for each handler needs to end in `/{time:%Y-%m-%d}` for a dated subfolder to be created.

**I created a discussion** at https://github.com/JVital2013/goestools-win/discussions/7 where folks can share custom goesproc configs, and I shared a copy of the default config with dated subfolders enabled. Just replace your config with the one from the thread to create the folders.

**As for "is this intentional":** yes, it is, but not for anything inherent to Goestools for Windows. Dated subfolders make Vitality GOES significantly slower, especially when hosted on Windows.  I also don't personally like saving data in dated subfolders - I think it makes it harder to browse data. But, I'm going to pretend the Vitality GOES thing is the main reason 🤣.
